lib/ical/event.js

```diff
@@ -452,6 +452,18 @@ ICAL.Event = (function() {
       this._setProp('description', value);
     },
 
+    /**
+     * The event color.
+     * @type {String}
```

Comment:
Since this is not an rfc5545 property let's mention where it comes from.

```suggestion
     * The event color from [rfc7986](https://datatracker.ietf.org/doc/html/rfc7986)
     * @type {String}
```